src/Nethermind/Nethermind.Config/BlocksConfig.cs

```diff
@@ -58,6 +58,7 @@ private static string GetDefaultVersionExtraData()
         public int PreWarmStateConcurrency { get; set; } = 0;
 
         public int BlockProductionTimeoutMs { get; set; } = 4_000;
+        public double SingleBlockImprovementOfSlot { get; set; } = 0.25;
```

Comment:
Given this is the default, now you need to add this field in every chain config file that is not PoS, (Taiko Alethia, Taiko Hekla, Optimism, Base, etc etc)

Reply:
Why?

Taiko doesn't use timeout anyway; as it doesn't improve blocks

Reply:
okay, ben brought up that taiko doesnt do block improvement which is true, and OP can be adjusted later